src/12_scopes.py

```diff
@@ -16,11 +17,12 @@ def change_x():
 # This nested function has a similar problem.
 
 def outer():
+    global y
     y = 120
 
     def inner():
+        global y
```

Comment:
how could you achieve this without using the global keyword twice?  There's another keyword that you would only need to use once.

Reply:
I couldn't find it :(